packages/salesforcedx-vscode-core/src/commands/templates/forceApexUnitClassCreate.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { LocalComponent } from '@salesforce/salesforcedx-utils-vscode';
+import { CompositeParametersGatherer, MetadataTypeGatherer, OverwriteComponentPrompt, SfdxCommandlet, SfdxWorkspaceChecker, SimpleGatherer } from '../util';
```

Comment:
Things like SfdxCommandlet exist both in utils and in `@salesforce/salesforcedx-utils-vscode`.  Made for some pain in the unit tests.